src/test/java/guitests/AssigneePickerTests.java

```diff
@@ -12,28 +12,51 @@
 
 public class AssigneePickerTests extends UITest {
 
-    private static final String TEXT_FIELD_ID = "#assigneePickerTextField";
-
     @Test
     public void showAssigneePicker_typeQuery_displaysCorrectly() {
         triggerAssigneePicker(getIssueCell(0, 9).getIssue());
         clickAssigneePickerTextField();
-        selectAll();
-        push(KeyCode.BACK_SPACE);
         type("world");
         assertEquals("world", getAssigneePickerTextField().getText());
         exitCleanly();
     }
 
     @Test
-    public void showAssigneePicker_noAssignee_assigneeAssigned() {
+    public void showAssigneePicker_pickAUser_userPickedAssigned() {
         TurboIssue issue = getIssueCell(0, 9).getIssue();
         triggerAssigneePicker(issue);
-        selectAll();
-        push(KeyCode.BACK_SPACE);
-        type("User");
+        type("User 1");
         push(KeyCode.ENTER);
+        assertEquals("User 1", issue.getAssignee().get());
+        exitCleanly();
```

Comment:
You can put a `@Before` for `exitCleanly()`; that means you don't have to manually call it in every `@Test` method.